packages/expo-sms/src/SMS.types.ts

```diff
@@ -1,3 +1,9 @@
 export type SMSResponse = {
   result: 'unknown' | 'sent' | 'cancelled';
 };
+
+export type SMSAttachment = {
+  uri: string;
+  mimeType: string;
```

Comment:
@EvanBacon - I ended up going with `mineType` here to match the parameter naming of `Sharing.shareAsync`'s parameter.